inc/class-wpseo-meta.php

```diff
@@ -439,12 +469,23 @@ public static function sanitize_post_meta( $meta_value, $meta_key ) {
 		$clean     = self::$defaults[ $meta_key ];
 
 		switch ( true ) {
+			case ( $meta_key === self::$meta_prefix . 'inclusive_language_score' ):
+			case ( $meta_key === self::$meta_prefix . 'content_score' ):
+			case ( $meta_key === self::$meta_prefix . 'estimated-reading-time-minutes' ):
 			case ( $meta_key === self::$meta_prefix . 'linkdex' ):
 				$int = WPSEO_Utils::validate_int( $meta_value );
 				if ( $int !== false && $int >= 0 ) {
 					$clean = strval( $int ); // Convert to string to make sure default check works.
 				}
 				break;
+			// Ids are always integers that are greater then 0.
+			case ( strpos( $meta_key, 'image-id' ) !== false ):
+			case ( strpos( $meta_key, self::$meta_prefix . 'primary' ) === 0 ):
+				$int = WPSEO_Utils::validate_int( $meta_value );
+				if ( $int !== false && $int > 0 ) {
+					$clean = strval( $int );
+				}
+				break;
```

Comment:
@igorschoester I decided also to add to the id's fields to the sanitization of the primary term id.

Reply:
Nice. I am wondering if it is a problem to have them here, while the check is less precise. Any chance of others hitting this check by accident?

Reply:
Good point 👍🏽